Doc/library/idle.rst

```diff
@@ -557,7 +557,7 @@ IDLE-console differences
 
 As much as possible, the result of executing Python code with IDLE is the
 same as executing the same code in a console window.  However, the different
-interface and operation occasionally affects visible results.  For instance,
+interface and operation occasionally affect visible results.  For instance,
```

Comment:
Too late ;-)  Since the intended meaning is the same either way, what matters here is which feels more 'right' for a fresh reader, such as you.